src/pages/index.js

```diff
@@ -36,7 +36,7 @@ export const query = graphql`
           title
           background {
             childImageSharp {
-              fluid(maxWidth: 300) {
+              fluid(maxWidth: 3000, quality: 70) {
```

Comment:
Wait actually, this is gonna affect Retina display quality noticeably, is it going to have a big impact if we just leave this at quality 100?